### PR TITLE
feat: support dib format for windows

### PR DESCRIPTION
**Pull request by pluveto**

### CODE REVIEW

File: clipboard.go [hunk 1]
```diff
@@ -79,6 +79,8 @@ const (
 	FmtText Format = iota
 	// FmtImage indicates image/png clipboard format
 	FmtImage
+	// CF_DIB bitmap image. Windows only
+	FmtImageDib
```
Comment (changkun) [inline clipboard.go:83]: I like this contribution. But despite we documented this format is windows only, we still need to support other platforms, including macOS and Linux.

**Review by changkun — COMMENTED** [review]
Adding a single format for a specific platform might out of the goal of this cross platform package. 

The general goal would allow users to add customized format handler, such as #17

File: clipboard.go (hunk 1 above)
Comment (changkun) [reply clipboard.go:83]: Another thought: if Dib is essentially an image, why not just merge it into FmtImage. If DIBV5 is not recognizable on windows, then we do another try on the DIB format. If it works, we convert the DIB into PNG and return the data.

Comment (pluveto) [reply clipboard.go:83]: > Another thought: if Dib is essentially an image, why not just merge it into FmtImage. If DIBV5 is not recognizable on windows, then we do another try on the DIB format. If it works, we convert the DIB into PNG and return the data.

Good idea. Updated.

Comment (changkun) [reply clipboard.go:83]: Thanks! Then this format declaration is not necessary anymore.

Comment (pluveto) [reply clipboard.go]: Updated

**Review by changkun — APPROVED** [review]
Thanks! and happy new year :)

**Pull request merged**